tests/TestCase.php

```diff
@@ -48,9 +48,9 @@ protected function getEnvironmentSetUp($app)
                 'driver' => 'pgsql',
                 'host' => 'testing',
                 'port' => '5432',
-                'database' => env('DB_DATABASE'),
-                'username' => env('DB_USERNAME'),
-                'password' => env('DB_PASSWORD'),
+                'database' => env('DB_DATABASE', 'laravel'),
```

Comment:
Why do you need to provide default value here?, should not be set in phpunit file?

Reply:
Yes, that's the second option. But no prob for me with default value too.